shell-completion/zsh/_systemctl.in

```diff
@@ -449,6 +449,7 @@ _arguments -s \
     {-l,--full}"[Don't ellipsize unit names on output]" \
     '--show-types[When showing sockets, show socket type]' \
     {-i,--ignore-inhibitors}'[When executing a job, ignore jobs dependencies]' \
+    '--obey-inhibitors[When executing a job, obey jobs dependencies]' \
```

Comment:
the comment here is actually wrong... probably copy/paste error from some other wsitch. should be fixed for both `--ignore-inhibitors` and `--objey-inhibitors`